module/Activity/src/Activity/Form/ActivitySignup.php

```diff
@@ -58,6 +65,23 @@ public function initialiseExternalForm($fields)
         $this->initialiseForm($fields);
     }
 
+    public function initialiseExternalForm($fields)
+    {
+        $this->add([
+            'name' => 'captcha',
+            'type' => 'Zend\Form\Element\Captcha',
+            'options' => [
+                //'label' => 'Beep-boop I\'m a robot.',
+                'captcha' => new ImageCaptcha([
+                    'font' => getcwd() . '/public/fonts/bitstream-vera/Vera.ttf',
```

Comment:
Why is `getcwd()` needed here? Wouldn't just using `public/fonts/bitstream-vera/Vera.ttf` work?